src/did/prefix.ts

```diff
@@ -31,21 +31,21 @@ export const parseMagicBytes = (prefixedKey: Uint8Array): {
   if (hasPrefix(prefixedKey, RSA_DID_PREFIX)) {
     return {
       keyBytes: prefixedKey.slice(RSA_DID_PREFIX.byteLength),
-      type: 'rsa'
+      type: "rsa"
     }
 
   // EDWARDS
   } else if (hasPrefix(prefixedKey, EDWARDS_DID_PREFIX)) {
     return {
       keyBytes: prefixedKey.slice(EDWARDS_DID_PREFIX.byteLength),
-      type: 'ed25519'
+      type: "ed25519"
     }
 
   // EDWARDS
```

Comment:
just saw this (it's my fault lol), this should be bls^^

Reply:
(the comment that is)

Reply:
Ah yeah right. Lemme fix that